### DISCUSSION TITLE
About computer configuration

### DISCUSSION THREAD

**Discussion by StacyIssac** [Svito-zar/gesticulator#23] [Q&A] [answered]
Hello, I would like to ask you about the computer configuration used in your trainning model and the approximate trainning time.

**Comment by Svito-zar** [comment, marked as answer]
Hi @StacyIssac .
The model was trained on a computer with 1 GPU, namely NVIDIA TITAN Xp. 
Each epoch took approximately 2.5m making 100 epochs of training to run for about 4h.
Did I answer your question?
